addons/mail/static/src/js/client_action.js

```diff
@@ -287,6 +356,41 @@ var ChatAction = Widget.extend(ControlPanelMixin, {
                 self.thread.scroll_to({offset: offset});
             });
     },
+    /**
+     * @private
+     */
+    _onModeration: function (messageID, decision) {
```

Comment:
should be in handlers section. also, signature is wrong, should accept a odooevent